### DISCUSSION TITLE
Pm2.5 in FDS

### DISCUSSION THREAD

**Discussion by ahmedahmed2056** [firemodels/fds#12835] [General]
Is there any way to model PM2.5 concentration from a burning material in FDS ?

**Comment by rmcdermo** [comment]
See user guide chapter on Aerosols.

**Reply by ahmedahmed2056** [reply]
The question was updated. I meant the  concentration of the PM2.5 not the Aerosol in general.

**Comment by mcgratta** [comment]
FDS cannot predict the generation of soot particulate. You would need to declare PM2.5 as a gas SPECies and specify its generation rate at a surface. You could also just specify a SOOT_YIELD and assume that all the soot is PM2.5

**Reply by ahmedahmed2056** [reply]
Thanks for your reply. So to define PM2.5 as a gas species, is there any reference or Scientific article that can be used for the properties needed to define it ? 

and after i define it as a species, how would I make it yield for my desired reaction ? make it complex stoichiometry reaction ?

**Comment by mcgratta** [comment]
You need to know what fraction of the total amount of soot is 2.5 microns in diameter or less. Then you can run FDS in default mode, using simple chemistry and a SOOT_YIELD, and then just use the given fraction of the total amount of soot.

**Reply by ahmedahmed2056** [reply]
I think this is hard to be known as soot is having many constituents and also the fact that pm2.5 is not coming only from soot. Thanks for your reply and have a nice day.

**Comment by mcgratta** [comment]
[Take a look at this report.](https://nvlpubs.nist.gov/nistpubs/Legacy/IR/nistir5273.pdf)

**Reply by ahmedahmed2056** [reply]
Thanks Dr Mcgratta that was really helpful. So can the same PM size distribution be taken for the case of vegetation too ?

**Comment by mcgratta** [comment]
I have no idea. Is crude oil the same as grass? Both are organic compounds, but that's about all I can say. I would guess that there is a fairly large community of researchers looking at global production and transport of pollutants from crop burning and wildfires. In fact, LMGTFY

![image](https://github.com/firemodels/fds/assets/11333911/cb93b8a6-d0f1-4c1d-b6e1-9405cc49ce74)
